content/en/agent/cluster_agent/clusterchecks.md

```diff
@@ -66,30 +87,28 @@ Enable the `clusterchecks` configuration provider on the Datadog **Node** Agent.
           polling: true
     ```
 
-[Restart the Agent][1] to apply the configuration change.
+[1]: /agent/cluster_agent/setup/
+{{% /tab %}}
+{{< /tabs >}}
 
-**Note**: The [Datadog Helm Chart][4] offers the possibility to deploy, through the `clusterChecksRunner` field, a set of Datadog Agents configured to run cluster checks only.
 
-### Custom checks
-
-Running [custom Agent checks][5] as cluster checks is supported, as long as all node-based Agents are able to run it. This means your checks' code:
+**Note**: When running Cluster Checks the metrics reported by the Agent are not linked to a given hostname. As these are meant to be cluster centric metrics, and not necessarily host based metrics. As a result these metrics will also not inherit any host-level tags associated with that host. Such as those inherited from a cloud provider or added by the Agent's `DD_TAGS` environment variable. To add tags to cluster check metrics use the `DD_CLUSTER_CHECKS_EXTRA_TAGS` environment variable.
```

Comment:
```suggestion
**Note**: With cluster checks, the metrics reported by the Agent are not linked to a given hostname because they are meant to be cluster-centric metrics and not necessarily host-based metrics. As a result, these metrics do not inherit any host-level tags associated with that host, such as those inherited from a cloud provider or added by the Agent's `DD_TAGS` environment variable. To add tags to cluster check metrics, use the `DD_CLUSTER_CHECKS_EXTRA_TAGS` environment variable.
```